CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(polylineencoder)
+set (CMAKE_CXX_STANDARD 11)
+
+add_library(encoder
+  src/polylineencoder.cpp
```

Comment:
Could you please add the corresponding header too?

Reply:
done